controlplane/kubeadm/controllers/controller_test.go

```diff
@@ -175,7 +175,7 @@ func TestReconcileUpdateObservedGeneration(t *testing.T) {
 		errGettingObject = testEnv.Get(ctx, util.ObjectKey(kcp), kcp)
 		g.Expect(errGettingObject).NotTo(HaveOccurred())
 		return kcp.Status.ObservedGeneration
-	}, 10*time.Second).Should(Equal(generation))
+	}, 20*time.Second).Should(Equal(generation))
```

Comment:
@relyt0925 let's this revert this given that #4466 has merged

Reply:
got it!